jhipster-jdl.jh

```diff
@@ -21,19 +21,19 @@ entity UserDetails {
     city String,
     country String,
     address String,
-    additionalInfo String
+    additionalInfo TextBlob
 }
 
 entity UserMails {
 	sentDate Instant required,
     sentBy String required,
-    mailType MailType,
```

Comment:
Let's get rid of the enum restriction on the mail Type here since the list keeps expending quite frequently

Reply:
Yeah that is good point. MailType will need more values for when we add company entity.